pom.xml

```diff
@@ -44,4 +44,16 @@
         <module>standalone</module>
         <module>tests</module>
     </modules>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.mule.runtime</groupId>
+                <artifactId>mule</artifactId>
+                <version>${project.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
```

Comment:
This will make the versions in Mule POM take precedence over any other versions in intermediate POM files, this is not how it works today and I'm not sure we want that.